app/src/main/java/com/hedvig/app/ui/compose/composables/screens/GenericInfoScreen.kt

```diff
@@ -0,0 +1,97 @@
+package com.hedvig.app.ui.compose.composables.screens
+
+import android.content.res.Configuration
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hedvig.app.R
+import com.hedvig.app.ui.compose.composables.buttons.LargeContainedButton
+import com.hedvig.app.ui.compose.composables.buttons.LargeOutlinedButton
+import com.hedvig.app.ui.compose.theme.HedvigTheme
+import com.hedvig.app.ui.compose.theme.HedvigTypography
+
+@Composable
+fun GenericInfoScreen(
+  modifier: Modifier = Modifier,
+  title: String,
+  description: String,
+  @DrawableRes icon: Int,
+  primaryButtonText: String = stringResource(R.string.continue_button),
+  onPrimaryButtonClicked: (() -> Unit)?,
+  secondaryButtonText: String = stringResource(R.string.home_tab_error_button_text),
+  onSecondaryButtonClicked: (() -> Unit)?,
+) {
+  Surface(
+    modifier = modifier.fillMaxSize(),
+    color = MaterialTheme.colors.onPrimary,
+  ) {
+    Column(modifier = Modifier.padding(16.dp)) {
+      Icon(
+        painter = painterResource(id = icon),
+        contentDescription = null,
+        Modifier.padding(top = 80.dp).height(25.dp).width(25.dp),
+      )
+      Text(
+        text = title,
+        style = HedvigTypography.h5,
+        modifier = Modifier.padding(top = 16.dp),
```

Comment:
This again is not *that* important and mostly a stylistic choice, but it could be that instead of having each Text specify its own custom padding to fit the design, this could instead be `Spacer` objects in-between those texts like `Spacer(Modifier.height(16.dp))`
The reason I usually say it is so that we can have an easier mental model that "here is a Column, it has a text, some space, a text, more space, and a text" instead of "here's a column with 3 texts, and let's look into their modifiers to figure out their placement rules".
Also comes with the exctra benefit that if some of the text moves around or gets deleted in some future design, the modifier that specified the spacings isn't gone with it, but it still lives in the Spacer. And then if you get a Spacer next to another Spacer you *know* it's time to either remove one of them or merge them. With the modifiers that's not that obvious.

Reply:
Yes that is a very good point.